pkg/instrumentation/dotnet.go

```diff
@@ -108,30 +102,20 @@ func injectDotNetSDK(logger logr.Logger, dotNetSpec v1alpha1.DotNet, pod corev1.
 			}},
 		})
 	}
-
-	pod.Spec.Containers[index] = container
-	return pod
+	return pod, true
 }
 
-func trySetEnvVar(logger logr.Logger, container *corev1.Container, envVarName string, envVarValue string, concatValues bool) bool {
+// set env var to the container.
+func setDotNetEnvVar(container *corev1.Container, envVarName string, envVarValue string, concatValues bool) {
```

Comment:
I suggest describing what `concatValues` is supposed to offer.
AFAIK it should be set to `true` if the env var supports multiple values supported by `:`. If it is set to `false`, the original container's env var value has priority.

Reply:
Done

Reply:
Feel free to resolve